build/MergeRules.ps1

```diff
@@ -0,0 +1,67 @@
+param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "full path to json rules file")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $mainFilePath,
+
+    [Parameter(Mandatory = $true, HelpMessage = "full path to json rules file to be merged")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $filesToMergePath,
+
+    [Parameter(Mandatory = $true, HelpMessage = "path to the result - merged file")]
+    [string]
+    $targetFilePath
+)
+
+function DecodeUtfCharacters($text)
+{
+    $charRegex = [regex]'\\u[a-zA-Z0-9]{4}'
+    $matches = $charRegex.Matches($text)
+
+    if (!$matches)
+    {
+        return $text
+    }
+
+    $utfToReplace = $matches | Select-Object -ExpandProperty Value -Unique
+
+    $result = $text
+    foreach ($u in $utfToReplace)
+    {
+        $repl = [regex]::Unescape($u)
+        $result = $result.Replace($u, $repl)
+    }
+
+    $result
+}
+
+
+$mainFileText = (Get-Content -Path $mainFilePath -Encoding UTF8) -Join "`n"
+$mainFileJson = ConvertFrom-Json -InputObject $mainFileText
+
+$fileToMergeText = (Get-Content -Path $fileToMergePath -Encoding UTF8) -Join "`n"
+$fileToMergeJson = ConvertFrom-Json -InputObject $fileToMergeText
+
+$result = $mainFileJson
```

Comment:
Indeed, strictly speaking that's not needed. But it makes it easier to debug and is semantically cleaner.